tesseract_environment/src/environment.cpp

```diff
@@ -1423,23 +1545,23 @@ bool Environment::applyAddCommand(AddLinkCommand::ConstPtr cmd)
 
   if (!link_exists && joint_exists)
   {
-    CONSOLE_BRIDGE_logWarn("Tried to add link (%s) which already exists with a joint provided which does not exist. "
+    CONSOLE_BRIDGE_logWarn("Tried to add link (%s) which does not exists with a joint provided which does not exist. "

```

Comment:
I think this is still wrong.  "Tried to add link (%s) which does not exist with a joint provided which **already** exists."

Reply:
Yep, fixed.